lib/fs/filesystem.js

```diff
@@ -194,10 +193,9 @@ class FileSystem {
 
   // eslint-disable-next-line no-unused-vars, class-methods-use-this
   createWriteStream(path, options) {
-    throw new Error('Not yet supported by API');
     // TODO: options is optional.
     // https://nodejs.org/api/fs.html#fs_fs_createwritestream_path_options
-    // return this.rest.upload(path);
+    return operationWrapper(this.rest, 'upload', [path]);
```

Comment:
Add callback here to populate cache with new item.

Reply:
Actually, I will make a branch `more-more-cache` to implement these, and I will implement more tests (focused on caching).